### ISSUE TITLE
Bad output for failing hasClasses call

### ISSUE THREAD

**Issue by jzaefferer** [arschmitz/qunit-assert-classes#5]
For the given failing assertion:

``` js
assert.hasClasses( tabs[ 0 ], "ui-tab" );
```

The output currently looks like this (at least the version included in the jQuery UI repo):

```
Element must have classes
Expected: ""
Result: "ui-tab"
```

The Expected and Result seem wrong, since the "ui-tab" class is expected, but missing. I also suggest using a more specific message when matching a single class, like "Element must have a class".

**Comment by arschmitz** [comment]
Have you tried this with version 0.1.6 or 0.1.7? The incorrect order should have been fixed in 0.1.6. 

as far as the message im thinking `Element must have class(es)` should be good enough. Actually changing the argument based on the length would require splitting and checking the passed class list in multiple places, so i would rather avoid it. Thoughts?

**Comment by jzaefferer** [comment]
I'd still prefer more precise output. You can use the return value to pass information about the expected value back to the individual assertions.
